test/fixture-scripts/index.ts

```diff
@@ -73,16 +71,6 @@ async function run() {
 
 let vscodeExecutablePath: string
 
-async function prepareVSCode() {
-  vscodeExecutablePath = await downloadAndUnzipVSCode('1.77.3')
-  const cliPath = resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath)
-
-  spawnSync(cliPath, ['--install-extension', 'vue.volar'], {
-    encoding: 'utf-8',
-    stdio: 'inherit',
-  })
-}
```

Comment:
Since we no longer need to pre-install this extension, we don't need any of this.